fix grammatical issues in what_is_accessibility (#23041)
mdn/content: files/en-us/learn/accessibility/what_is_accessibility/index.md

Accessibility is the right thing to do. Providing accessible sites is part of the law in some countries, which can open up some significant markets that otherwise would not be able to use your services or buy your products.

> Building accessible sites benefit everyone:

Reviewer: Original is correct

A good foundation of accessibility for people with cognitive impairments includes:

- Delivering content in more than one way, such as by text-to-speech or by video.

> - Easily understood the content, such as text written using plain-language standards.

Reviewer: Original is correct. "Easily understood" is a modifier for "content".

Author: Yes, now I understood it. Will undo the change.

- Is my date picker widget usable by people using screen readers?
- If content updates dynamically, do visually impaired people know about it?

> - Are my UI buttons accessible using the keyboard and on-touch interfaces?

Reviewer: Original is correct. If you find it weird to join an adverbial phrase with a prepositional phrase, consider this:

```suggestion
- Are my UI buttons accessible to both keyboard and touch interface users?
```

Author: Thank you for the suggestion.
